docs/concepts/assets/transfers.md

```diff
@@ -2,45 +2,86 @@
 * How much is available to transfer
 * The starting epoch for the transfer
-* Optionally, the end epoch when the transfers should stop. If it's not specified, the transfer run until cancelled
+* Optional: the end epoch when the transfers should stop. If it's not specified, the transfer run until cancelled
 * The percentage of the full amount to pay each epoch, which is defined using the factor - a decimal
   - The amount paid at the end of each epoch is calculated using the following formula: `amount = start amount x factor ^ (current epoch - start epoch)`
+* Optional: When used to fund a reward pool, the [distribution method](../trading-on-vega/fees-rewards.md#how-rewards-are-scaled) - pro-rata or based on rank
 
-#### Recurring transfer limits
+### Recurring transfer limits
 While a party (public key) can have multiple transfers set up to move assets to different accounts, each party can only have one recurring transfer between two given accounts at the same time. For example, a party can transfer from their general account to Public Key A and Public Key B, but they cannot set up two recurring transfers of different amounts both going to Public Key B.
 
-### Cancel or amend transfers
-It's possible to cancel a recurring transfer, but not to amend. If you want to change your transfer, you'll need to cancel the existing transfer and submit a new one.
+## Cancel or amend transfers
+It's possible to cancel a recurring transfer, but not to amend. If you want to change your transfer, you'll need to cancel the existing transfer and submit a new one. Transfers initiated by governance will need to be cancelled by submitting a proposal to cancel the transfer.
 
 If the asset used to fund a recurring transfer is depleted, either because the funds have run out or it's less than the <NetworkParameter frontMatter={frontMatter} param="transfer.minTransferQuantumMultiple" />` x quantum`, then the transfer is cancelled automatically. You'll have to set up a new transfer if you want to keep funding the key/account.
+
+## Governance-initiated transfers
+Assets being moved out of certain accounts requires community support, through a governance proposal and vote. Generally speaking, they're accounts that have assets moved into them after markets are settled, because of market protection movements, or entirely funded by community members that transfer assets into them. 
+
+The proposals give community members a chance to determine what they think the assets should be spent on, whether that's to fund [trading or validator rewards](../trading-on-vega/fees-rewards.md#trading-rewards), to move money from [insurance pools](./accounts.md#insurance-pool-accounts) into [network treasury accounts](./accounts.md#network-treasury-accounts), or for other purposes.
+
+These governance-initiated transfers can be one-off or recurring. A recurring transfer can only be cancelled when a governance proposal to cancel it is submitted and passes the governance vote.
```

Comment:
I don't know if this is the place or how is best to express this but you want to say somewhere maybe here that not all of the below are supported for one off gov transfers (all metric based rewards)